e2e/node/request-metadata.test.ts

```diff
@@ -0,0 +1,263 @@
+//
+// Copyright Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import {
+  afterAll,
+  beforeAll,
+  describe,
+  expect,
+  it,
+  test,
+  jest,
+} from "@jest/globals";
+
+import type { Session } from "@inrupt/solid-client-authn-node";
+import {
+  getNodeTestingEnvironment,
+  getPodRoot,
+  getAuthenticatedSession,
+} from "@inrupt/internal-test-env";
+import { getSolidDataset } from "../../src";
+
+const { Headers } = global;
```

Comment:
```suggestion
```

Reply:
Weird, for some reason when I was testing I had a `Headers is not defined` error, which was fixed by this, but I can't reproduce now even with this gone. Removed in ecf655c.